docs/guides/integration-javascript-client-side.md

```diff
@@ -281,7 +281,19 @@ In some cases, the user's DII is not available on page load, and getting the DII
 You can potentially avoid that cost by checking for an existing token that you can use or refresh. To do this, call
 [__uid2.isLoginRequired](../sdks/sdk-ref-javascript#isloginrequired-boolean) which returns a Boolean value. If it returns `true`, this means that the UID2 SDK cannot create a new advertising token with the existing resource and DII is required to generate a brand new UID2 token.
 
-The following code snippet demonstrates how you might integrate with the UID2 SDK for JavaScript for the two scenarios above&#8212;starting with no token as well as reusing/refreshing any existing UID2 token if found. 
+The only exception is if the following method returns true, 
```

Comment:
alternatively we can just say:

_The only exception is the DII could be opted out of UID2 and no identity/token should be generated for it. The UID2 SDK for JavaScript will respect the opt out preference and not generate UID2 tokens even if you call any of the `setIdentity` method calls with DII again._

And don't mention the ```__uid2.hasOptedOut()``` method and leave it be.

Reply:
I just tried this at https://euid.eu/docs/guides/integration-javascript-client-side using optout@example.com (which is opted out). I know this is EUID and not UID2 - I'm not sure if the behaviour is different. I'll try with our UID2 example site shortly.

![image](https://github.com/user-attachments/assets/21ed691f-25fc-4bca-b34c-486af06a722c)

It looks like nothing works as expected if the user has opted out :(

Reply:
I think isLoginRequired should return false if we have an optout identity. The user is logged in - they're opted out.